tools/ci_build/github/linux/docker/scripts/install_deps.sh

```diff
@@ -28,7 +28,7 @@ fi
 if [ "$INSTALLED_PYTHON_VERSION" = "3.4" ];then
   echo "Python 3.5 and above is needed for running onnx tests!" 1>&2
 else
-  source /tmp/scripts/install_onnx.sh $INSTALLED_PYTHON_VERSION
+  source ${0/install_deps/install_onnx} $INSTALLED_PYTHON_VERSION
```

Comment:
What if the current dir name contains 'install_deps' ?

Reply:
Good point - add suffix ".sh" as part of matching.

Reply:
Then, what if the current dir name contains 'install_deps.sh' ?

Reply:
Then let's match only the end.

Reply:
Can you try [this solution](https://stackoverflow.com/questions/59895/get-the-source-directory-of-a-bash-script-from-within-the-script-itself)?

Reply:
![image](https://user-images.githubusercontent.com/48490400/59525536-740b3280-8e8b-11e9-8b9a-04f869cf8179.png)

Reply:
Linux pipeline says everything.
Could u please try with your docker file to double check?